### DISCUSSION TITLE
Drag and drop from one component to a second one

### DISCUSSION THREAD

**Discussion by derbrill** [lowcoder-org/lowcoder#1355] [Q&A]
Hi all, please excuse my ignorance. I am just evaluating lowCoder and am pretty impressed.

I have to build an app featuring two trees with different structures where I would like to "link" a node or field in component one to a node or field in component two by drag and drop. Also I would like to be able to create an object that describes the relations between tree one and tree 2, being able to add other fields and or nodes in the document from other parts of the UI. I hope this explanation makes sense.

Would something like this be possible, or is there even a component that does this already? Or somebody available for hire that could help build such a thing?

All the best,

Malte

**Comment by FalkWolsky** [comment]
We do have the transfer component, that is close to your needs. But we haven't added yet full support for drag n drop. Please excuse!
